src/js/components/Drop/Drop.js

```diff
@@ -24,13 +30,15 @@ const Drop = forwardRef(
     useEffect(() => setOriginalFocusedElement(document.activeElement), []);
     const [dropContainer, setDropContainer] = useState();
     const containerTarget = useContext(ContainerTargetContext);
-    useEffect(
-      () =>
+    const containerChildNodesLenght = useRef(null);
```

Comment:
Why does this need to be a ref? I think it could just be a regular variable. Also can we rename to `containerChildNodesLength` and add a comment about why we need this check for the Tip component

Reply:
Hi jcfilben, thanks for your feedback!

I added a comment and changed the misspelled variable name.

Unfortunately using a variable will not resolve this: 

- if we define the variable inside the useEffect and then assign it inside the if condition, the issue will not be solved when the component is destroyed and then re-created, as everytime the useEffect runs,  the variable will be initialised again and the if condition will always be true.
- if we define the variable outside the useEffect and assign a value inside, it will lose its assignment after each render which will fix this issue as long as the drop component is without any side-effects.(A tooltip generating plain text will work fine). 
In a bit more complex case in which the parent element of the Drop component has sideEffects that updates the dropped element (i.e. a Calendar), the variable will be re-initialised and we will incur in the bug again.

I would prefer useRef as it is guaranteed not to change between render(s)

Reply:
Thanks for the explanation, that makes sense